app/middlewares/validateToken.js

```diff
@@ -19,6 +19,11 @@ exports.validateToken = (req, res, next) => {
     if (!user) {
       throw authenticationError('User not found');
     }
+
+    if (moment(token.exp).isBefore(new Date())) {
```

Comment:
[jwt-simple](https://github.com/hokaccha/node-jwt-simple/blob/master/lib/jwt.js#L101) already checks this